src/app/shared/admin/admin-candidates/admin-elections-candidate-modal.component.ts

```diff
@@ -120,6 +143,8 @@ export class AdminCandidatesRowComponent implements OnInit {
           (data) => {
           this.rowData = Object.assign({}, data);
           this.rowFormGroup.markAsPristine();
+          this.notSaved.emit(false);
```

Comment:
can we see if there's a way use the state of the form group (pristine/dirty/etc) to determine whether to allow adding a new candidate?

Reply:
I don't think we're using a form group for this part of the code